crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -58,6 +58,13 @@ impl<'w, 's> Commands<'w, 's> {
         }
     }
 
+    /// Returns an [EntityCommands] for the given `entity` (if it exists) or spawns one if it doesn't exist.
+    /// This will return [None] if the `entity` exists with a different generation.
+    ///
+    /// # Note
```

Comment:
This note should really be duplicated on Entity::new if we're keeping it around.